specs/template.md

```diff
@@ -105,7 +111,7 @@
 
 ### Test Plan
 
-*What is the plan for testing the component, if different from the normal path?*
+*What is the plan for testing the component, if different from the normal path? Note that the normal plan includes unit tests for basic state/behavior as well as e2e tests to validate the specific user stories described above.*
```

Comment:
I tend to not use abbreviations since I suspect most people understand e2e, however, some may not so I'd recommend spelling it out.